### ISSUE TITLE
Ensure link text is meaningful within context

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2704]
**OCIO Violation**: Ensure link text is meaningful within context
**Severity**: 6
**Occurrence:** 15%
**Priority:** 6.7
**Note:** This A element does not have a mechanism that allows an accessible name value to be calculated

**Instances:**

**Page:** https://onrr.gov/references/frns
**Violation Description:** a href="https://www.federalregister.gov/documents/2022/04/04/2022-06640/major-portion-prices-and-due-date-for-additional-royalty-payments-on-gas-produced-from-indian-lands"/></a/

**Page:** https://onrr.gov/references/valuation?tabs=pricing,indian-gas-index-zone
**Violation Description:** a href="https://www.ecfr.gov/current/title-30/chapter-xii/subchapter-a/part-1206" target="_blank"/></a/

**Page:** https://onrr.gov/references/handbooks/geothermal-class-1
**Violation Description:** a target="_blank" href="/document/class-1-handbook.pdf" data-v-d132c190=""/></a/

**Page:** https://onrr.gov/references/handbooks/geothermal-class-2-and-3
**Violation Description:** a target="_blank" href="/document/class-2-3-handbook.pdf" data-v-d132c190=""/></a/

**Page:** https://onrr.gov/references/handbooks/minerals-revenue-reporter-handbook
**Violation Description:** a target="_blank" href="/document/rrm-printable.minerals.revenue.handbook.pdf" data-v-d132c190=""/></a/

**Comment by greensomes** [comment]
@eelzi-ONRR, I think I've resolved all of the exceptions above in both the dev and prod environments. Is there a way to run a scan again to make sure these have fallen off?

**Comment by eelzi-ONRR** [comment]
@greensomes Almost but one passed.

This one still fails in both dev and prod. ANDI showed me there's a second link in this link, and that's what's causing the failure. I can see it in the code too: 

Page: https://onrr.gov/references/valuation?tabs=pricing,indian-gas-index-zone
Violation Description: a href="https://www.ecfr.gov/current/title-30/chapter-xii/subchapter-a/part-1206" target="_blank"/></a/

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbHd4IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--bab4c3d315a2affd71e0af0395263a1ca0eaf00d/image.png)

**Comment by greensomes** [comment]
@eelzi-ONRR, K! It's fixed!

**Comment by eelzi-ONRR** [comment]
@greensomes it passes now! Thanks!
